docs/contributing/publishing-a-pre-release.md

```diff
@@ -0,0 +1,21 @@
+# Publishing pre-release of GOV.UK Frontend
+
+We use pre-releases when:
+- We are working on a new contribution and want to see it in the Design System website
+- We want to try an experimental feature as if it's pushed to npm.
+
+This is done by pushing a release to a new branch which can be installed by npm.
```

Comment:
It might be obvious to some but should this make it clear that this is not an actual release but rather the equivalent of the files that get pushed to npm? 

Maybe something like `pushing the files used for a GOV.UK Frontend release (the contents of the `package` directory) to a new branch which can be installed by npm as if it was a released npm package. No changes get published to npm as part of the process.`

Reply:
Yeah doesnt hurt to be more explicit since this is a bit of a weird process, I'll update.